zipline/data/bundles/quandl.py

```diff
@@ -216,14 +223,27 @@ def quandl_bundle(environ,
     )
 
     raw_data.reset_index(inplace=True)
-
+    raw_data['symbol'] = raw_data['symbol'].astype('category')
+    raw_data['sid'] = raw_data.symbol.cat.codes
     adjustment_writer.write(
         splits=parse_splits(
-            raw_data[['symbol', 'date', 'split_ratio']],
+            pd.DataFrame(
```

Comment:
I think the inner `loc` should return a DataFrame in the calls to `parse_splits` and `parse_dividends` below